.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -2984,6 +2984,29 @@ s3_Scenario_ObjectLock:
             - description: A wrapper class for S3 functions.
               snippet_tags:
                 - S3LockWorkflow.dotnetv3.S3ActionsWrapper
+    JavaScript:

```

Comment:
I see the scenario metadata, but where are the tags for the rest of the snippets that are associated with the actions?

![image](https://github.com/awsdocs/aws-doc-sdk-examples/assets/110480692/c3988343-1f8c-469d-989e-f8dc99cd7d9d)

Reply:
Ah, yes. I should add those :)

Reply:
Metadata added.